docs/src/pages/[platform]/getting-started/introduction/index.page.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Introduction
+description: What is Amplify UI?
+supportedFrameworks: all
+---
+
+import { Alert } from '@aws-amplify/ui-react';
+
+Amplify UI is an open-source UI library that brings the simplicity and extensibility of AWS Amplify to UI development. It consists of connected components that simplify complex workflows like authentication and dynamic data, primitive components that form the building blocks of a UI, and themes to make Amplify UI fit any brand. **Extensibility and customization** are at the forefront of Amplify UI allowing easy to integration into any application regardless of the front-end tech stack. 
+
+We want to take care of the details, like accessibility and cloud connectivity, so developers can focus on their product.
+
+## Connected components
+
+Amplify UI connected components abstract away complex front-end code like authentication and dynamic data to provide an intuitive API. Connected components are built on top of primitive components and Amplify Libraries. 
+
+Connected components are completely customizable at every layer. Customize the look-and-feel with theming controls, override components and function calls, bring your own UI with a headless mode, or even go un-styled, giving you full control over state, layout, styling, and transitions.
+
+## Primitive components
+
+<Alert variation="info">
+  Primitive components are React-only for now
+</Alert>
+
+Amplify UI primitive components are in the middle of the spectrum between 'headless' UI frameworks like [Radix](https://www.radix-ui.com/) and [Headless UI](https://headlessui.dev/), and 'batteries included' frameworks like [MUI](https://mui.com/) and [Chakra](https://chakra-ui.com/). We want to provide a solid base that doesn't force a particular front-end architecture and every detail can be customized.
+
+All the styling is handled by plain CSS and theme-able with CSS variables. However we do provide some extra styling utilities in React like responsive style props, ThemeProvider and useTheme hook, and useBreakpointValue for responsive values.
```

Comment:
Perhaps we should have internal links to some of these styling utilities?
- [ThemeProvider](https://ui.docs.amplify.aws/react/theming/dark-mode#themeprovider)
- [useThemeHook](https://ui.docs.amplify.aws/react/theming#breakpoints:~:text=//%20Option%202%3A%20Get%20the%20theme%20object%20through%20the%20useTheme%20hook%20and%20style%20components%20with%20it)
- [useBreakPointValue](https://ui.docs.amplify.aws/react/theming/responsive#usebreakpointvalue)